src/txkube/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright Least Authority Enterprises.
+# See LICENSE for details.
+
+"""
+A Kubernetes client.
+"""
+
+__all__ = [
+    "version",
+    "IKubernetesClient",
+    "network_client", "memory_client",
+]
+
+from incremental import Version
```

Comment:
❓ `incremental` should probably be listed as a direct dependency in setup.py then.